ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -50,3 +50,45 @@ def test_numpy_nanpercentile(
         test_flags=test_flags,
         input_dtypes=input_dtypes,
     )
+
+# quantile
+@handle_frontend_test(
+    fn_tree="ivy.quantile",
+    dtype_values_axis=_statistical_dtype_values(function="quantile"),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_quantile(
+    dtype_values_axis,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    keep_dims,
+):
+    input_dtypes, values, axis = dtype_values_axis
+    if isinstance(axis, tuple):
+        axis = axis[0]
+
+    where, input_dtypes, test_flags = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtypes,
+        test_flags=test_flags,
+    )
+
+    for interpolation in ["linear", "lower", "higher", "midpoint", "nearest"]:

```

Comment:
This is not the ideal way to test each of these cases. Please refer to [test_correlating.py](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_correlating.py) where you can see an example usage of `st.sampled_from()`